h/views/admin/groups.py

```diff
@@ -46,24 +46,32 @@ def groups_index(context, request):
     renderer="h:templates/admin/groups_create.html.jinja2",
     permission="admin_groups",
 )
-class GroupCreateController(object):
+class GroupCreateView(object):
+    """View handlers for creating new groups"""
+
     def __init__(self, request):
-        user_svc = request.find_service(name="user")
-        list_org_svc = request.find_service(name="list_organizations")
-        self.organizations = {o.pubid: o for o in list_org_svc.organizations()}
+        self.request = request
+
+        self.user_svc = self.request.find_service(name="user")
+        self.list_org_svc = self.request.find_service(name="list_organizations")
+        self.group_create_svc = self.request.find_service(name="group_create")
+        self.group_members_svc = self.request.find_service(name="group_members")
+
+        self.organizations = {o.pubid: o for o in self.list_org_svc.organizations()}
+        self.default_org_id = Organization.default(self.request.db).pubid
+
         self.schema = CreateAdminGroupSchema().bind(
-            request=request, organizations=self.organizations, user_svc=user_svc
+            request=request, organizations=self.organizations, user_svc=self.user_svc
         )
-        self.request = request
         self.form = _create_form(self.request, self.schema, (_("Create New Group"),))
 
     @view_config(request_method="GET")
     def get(self):
-
+        """Render create-group form"""
```

Comment:
```suggestion
         """Render the admin create group form."""
```

Nice to have a docstring though!